content/articles/2018-10-23-dplyr-0-8-0-release-candidate.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: dplyr 0.8.0 release candidate
+author: Romain François
+date: '2018-10-23'
+slug: dplyr-0-8-0-release-candidate
+description: > 
+  dplyr 0.8.0 release candidate
+categories:
+  - package
+tags:
+  - dplyr
+  - tidyverse
+---
+
+```{r, echo = FALSE, message = FALSE}
+knitr::opts_chunk$set(collapse = T, comment = "#>")
+options(tibble.print_min = 4L, tibble.print_max = 4L)
+library(dplyr)
+```
+
+# Tidy grouping structure
```

Comment:
I think this should come at the end since it's mostly useful for more advanced dplyr users.